### ISSUE TITLE
Passkeys and Email Verification to login? Why?

### ISSUE THREAD

**Issue by carkks** [ente-io/ente#3221]
I've been sorting out the TOTP codes for my mother and I thought, if I activate the passkeys and the verification code by email, it will make it easier to access the auth.
If I lost the email, I could access it with the password and passkey.
If I lost the passkey, I could access with the verification code and the password.
But when I ran the test, it asked for both the verification code and the passkey, and to make matters “worse” (just inconvenience, the password is important for decryption) it asked for the password.
In the security tab, there could be an option under passkeys to prefer it (if enabled) when logging in. It's also inconvenient to have to confirm 3 times that the account belongs to you. Just two is fine.

**Comment by elouangrimm** [comment]
Yeah, I just had to factory reset my phone and I forgot to backup my key and I don't have a passkey for obvious reasons, and now I have lost access to like 15 accounts. Thanks Ente. I wish I could get a email confirmation or something.

**Comment by mrkdilkington** [comment]
I’ve noticed the same issue. Please let the user decide whether they want double or triple factor authentication when both email verification and passkeys are enabled. If they choose double, when you login you should get to choose your 2FA method. Just like Bitwarden, Google, or any other service that has email verification and passkeys. Please give this some attention! Thanks

**Comment by cotton7581** [comment]
I would like to log in to a new device using my password and passkey/security key without requiring access to my email account.

Requiring an email OTP creates an unnecessary dependency.

**Comment by AmanRajSinghMourya** [comment]
Thanks for raising this. Allowing users to choose between enabled authentication factors, or allowing a passkey to replace email verification, is a product and account-security decision rather than an application bug.

Moving this to Discussions → Enhancements. Related requests are also being discussed here:

https://github.com/ente/ente/discussions/8069
https://github.com/ente/ente/discussions/3804
